channels/telegram/src/main/kotlin/com/justai/jaicf/channel/telegram/TelegramReactions.kt

```diff
@@ -18,11 +19,34 @@ class TelegramReactions(
 ) : Reactions() {
 
     val chatId = request.chatId
+    private val messages = mutableListOf<Message>()
```

Comment:
should we really store all previous responses instead of the last one?

Reply:
@nikvoloshin nope, we shouldn't. But this is the way to edit any of sent message once we've started to use editors.